Makefile

```diff
@@ -17,6 +18,25 @@ release:
 		aws s3 cp build/$$os/layer0_$(L0_VERSION)_$$os.zip s3://xfra-layer0/release/$(L0_VERSION)/layer0_$(L0_VERSION)_$$os.zip ; \
 	done
 
+github-release:
```

Comment:
If we are to keep the Github binaries, this should run at the end of the existing `release` target.  
This can be accomplished in one or two lines, because all of the binaries exist in `layer0/build` by the end of that target.